### PR TITLE
Add Unit Test - Bootstrap Jest Usage

### PR DESCRIPTION
**Pull request by fcpranav**
This PR adds a sample unit test to the repo. The main idea is to get our testing framework in place. There will be subsequent PRs to increase test coverage as needed.

### CODE REVIEW

File: PlateUp-reactnative/constants/toast.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import {
+    AlertIOS, Platform, StatusBar, ToastAndroid
+  } from 'react-native';
+  
+  export const StatusHeight = StatusBar.currentHeight;
+  export const HeaderHeight = ( 3.5 + (StatusHeight || 0));
+  export const iPhoneX = () => Platform.OS === 'ios' && (height === 812 || width === 812);
```
Comment (eanile) [inline PlateUp-reactnative/constants/toast.js]: None of these 3 exports are relevant to toast. Is there really no way to have one util file? I don't like having an entire file just for a toast function when its really a utility function.

File: PlateUp-reactnative/constants/utils.js [hunk 2]
```diff
@@ -1,26 +1,22 @@
...
-export const HeaderHeight = (theme.SIZES.BASE * 3.5 + (StatusHeight || 0));
-export const iPhoneX = () => Platform.OS === 'ios' && (height === 812 || width === 812);
-
-export function toast(msg) {
-  if (Platform.OS === 'android') {
-    ToastAndroid.show(msg, ToastAndroid.LONG);
-  } else {
-    AlertIOS.alert(msg);
+  for (const param in params) {
+    if (params[param]) {
```
Comment (eanile) [inline PlateUp-reactnative/constants/utils.js:19]: What's going on here? Why are we checking params[param]?

File: PlateUp-reactnative/constants/utils.js [hunk 3]
```diff
@@ -1,26 +1,22 @@
...
 
-export function constructQueryParams(queryParams, newParam) {
-  if (queryParams) {
-    queryParams += (`&${newParam}`);
+function appendQueryParam(existingQueryParams, newParam) {
+  if (existingQueryParams) {
+    existingQueryParams += (`&${newParam}`);
   } else {
-    queryParams = (`?${newParam}`);
+    existingQueryParams = (`?${newParam}`);
```
Comment (eanile) [inline PlateUp-reactnative/constants/utils.js]: Is it necessary to have these parentheses around the strings?

File: PlateUp-reactnative/constants/utils.test.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
+import { constructQueryParams } from "./utils";
+
+
+describe('constructQueryParams', () => {
```
Comment (eanile) [inline PlateUp-reactnative/constants/utils.test.js:4]: Can we add a test where some params are empty and others aren't?

File: PlateUp-reactnative/constants/utils.test.js [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+            param1: "key1",
+            param2: "key2"
+        }
+
+        const queryParams = constructQueryParams(params, keyTransforms);
+
+        expect(queryParams).toEqual("")
+    });
+
+})
```
Comment (eanile) [inline PlateUp-reactnative/constants/utils.test.js:68]: Why is the test for the utils in the constants folder? Shouldn't we have a tess folder where all tests reside?

File: PlateUp-reactnative/constants/toast.js (hunk 1 above)
Comment (fcpranav) [reply PlateUp-reactnative/constants/toast.js]: It is specifically the `theme` import. If we can get rid of that, I'm sure it'll work.

File: PlateUp-reactnative/constants/utils.js (hunk 2 above)
Comment (fcpranav) [reply PlateUp-reactnative/constants/utils.js:19]: This is to filter for empty values. For instance, the filters will have an empty string if they aren't being used. We don't want to consider empty values for queryParams.

File: PlateUp-reactnative/constants/utils.js (hunk 3 above)
Comment (fcpranav) [reply PlateUp-reactnative/constants/utils.js]: Nah, can be removed.

File: PlateUp-reactnative/constants/utils.test.js (hunk 4 above)
Comment (fcpranav) [reply PlateUp-reactnative/constants/utils.test.js:4]: Sure

File: PlateUp-reactnative/constants/utils.test.js (hunk 5 above)
Comment (fcpranav) [reply PlateUp-reactnative/constants/utils.test.js:68]: There's different patterns for where tests live. For the frontend repos I've worked in, you have the test live alongside the original file.

File: PlateUp-reactnative/constants/toast.js (hunk 1 above)
Comment (fcpranav) [reply PlateUp-reactnative/constants/toast.js]: Actually, we realized that the complications aren't actually there, we can move everything back into utils!!!

**Review by eanile — APPROVED** [review]
